www/components/Participant.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Chip from "./Chip";
+import styles from "../styles/Participant.module.css";
+
+function Participant({ text, owner }) {
+  return (
+    <div>
+      <Chip className={styles.text} text={text} editing={true}
+        deleteFunc={true} icon={true}/>
```

Comment:
When there are lots of props do something like this:


```
<Chip
  className={styles.text}
  text={text}
  editing={true}
  deleteFunc={true}
  icon={true}
/>
```

Reply:
Just saying, prettier did that for me

Reply:
But you just had to replace with BS lint